### PR TITLE
update recipe

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@isuruf if you have a few minutes yo spare can you take a quick look at the Windows failures here. I'm completely lost b/c this used to work :-/

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,24 +15,24 @@ source:
   sha256: 6d60e03e1abd30a7d4afe547d157ce3dd7a8c166fc5e407fd6d62ae99ff30460
 
 build:
-  number: 0
+  number: 1
```
Comment (isuruf) [inline recipe/meta.yaml]: You need `merge_build_host: True   # [win]` here

Comment (ocefpaf) [reply recipe/meta.yaml]: Thanks! I'll try to submit that to the `conda smithy update-cb3`.

**Pull request merged**